fe/src/test/java/org/apache/doris/utframe/AnotherDemoTest.java

```diff
@@ -111,19 +114,25 @@ public static void beforeClass() throws EnvVarNotSetException, IOException,
         Thread.sleep(5000);
     }
 
-    // generate all port from between 20000 ~ 30000
-    private static void getRandomPort() {
-        Random r = new Random(System.currentTimeMillis());
-        int basePort = 20000 + r.nextInt(9000);
-        fe_http_port = basePort + 1;
-        fe_rpc_port = basePort + 2;
-        fe_query_port = basePort + 3;
-        fe_edit_log_port = basePort + 4;
-
-        be_heartbeat_port = basePort + 5;
-        be_thrift_port = basePort + 6;
-        be_brpc_port = basePort + 7;
-        be_http_port = basePort + 8;
+    @AfterClass
+    public static void TearDown() {
+        try {
+            FileUtils.deleteDirectory(new File(runningDirBase));
+        } catch (IOException e) {
+        }
+    }
+
+    // generate all port from valid ports
+    private static void getPorts() {
```

Comment:
We could move this method to `UtFrameUtils` ?

Reply:
those ports are class member of DemoTest, and getValidport method already in UtFrameUtils